src/pages/index.js

```diff
@@ -67,20 +136,19 @@ const editProfileFormValidator = new FormValidator(
   editProfileFormElement
 );
 const editProfileForm = new PopupWithForm(editProfileObject);
-const cardSelection = new Section(
-  {
-    renderer: (item) => {
-      const cardElement = createCard(item);
-      cardSelection.addItem(cardElement);
-    },
-    initialCards,
-  },
-  selectors.CardSelection
+let cardSelection;
+const deleteCardForm = new DeleteConfirmationPopup(deleteCardObject);
+const editProfileAvatarForm = new PopupWithForm(editAvatarObject);
+const editAvatarFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```